src/transformStateWithClones.js

```diff
@@ -7,7 +7,58 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let newObj;
```

Comment:
So if a newObj is an array, than there is no need in using `let` keyword, `const` would be enough

Reply:
Second, if it is an array, it's not only one new object, it's a lot of `states`, something like `stateHistory` would be great